.github/workflows/code-review.yml

```diff
@@ -0,0 +1,92 @@
+name: "Code Review"
+
+on:
+  push:
```

Comment:
Do we want to run the code review after merging PRs?

Reply:
I found a template that did this and I think it is a good idea to get a code review also on master branch after merge. It's not really necessary, but sometimes it happened that some changes went to master and broke it.

Reply:
You're right but usually the review is done before merging to master in order to *never* break it

Reply:
Yeah, it was just a nitpicky check. I removed it.